search/result.cpp

```diff
@@ -64,6 +66,24 @@ uint32_t Result::GetFeatureType() const
   return m_featureType;
 }
 
+std::string Result::GetReadableFeatureType() const
```

Comment:
This function is used only once. Isn't it better to inline it?

Reply:
What do you mean to "inline it"?

Reply:
> What do you mean to "inline it"?

To copy the content of the function to the location where it's being used.

But I don't think it's a good idea.

Reply:
@AndrewShkrob what are pros of leaving it? I've listed cons below.

Reply:
Inlined